cmd/test/cmd.go

```diff
@@ -308,12 +308,11 @@ func doRun(ctx context.Context, servicesToTest []string, options *Options, ioCtr
 		if err := manifest.ExpandEnvVars(); err != nil {
 			return analytics.TestMetadata{}, fmt.Errorf("failed to expand manifest environment variables: %w", err)
 		}
-		oktetoLog.Information("'%s' was already deployed. To redeploy run 'okteto deploy'", devenvName)
 	}
 
 	metadata := analytics.TestMetadata{
 		StagesCount: len(testServices),
-		WasDeployed: needsDeploy,
+		WasDeployed: wasDeployed,
```

Comment:
I would track if the `--deploy` flag is used and remove the computation of `wasDeployed`

Reply:
@pchico83 any objection in renaming it to `Deployed`, then? so that we can keep the name consisent and leave `WasDeployed` available in the future, in case we want to track "re-deploys".

Reply:
Yes, `deployed` sounds good

Reply:
Changed in 34441f69

Reply:
Changed in 27cc3316